src/Prometheus/Collector.php

```diff
@@ -13,13 +13,15 @@ abstract class Collector
     protected $name;
     protected $help;
     protected $labels;
+    protected $defaultLabels;
 
     /**
      * @param Adapter $storageAdapter
      * @param string $namespace
      * @param string $name
      * @param string $help
      * @param array $labels
+     * @param array $defaultLabels
      */
     public function __construct(Adapter $storageAdapter, $namespace, $name, $help, $labels = array())
```

Comment:
Change `$labels = array()` to `$labels = []` to be consistent across all declarations.

Reply:
Common declaration in this repo is `array()`. It'll need to change everywhere